net-proxy/proxychains_ng/proxychains_ng-4.14.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A tool to use SOCKS or HTTP proxy with programs that don't support it"
+DESCRIPTION="
+  ProxyChains is a UNIX program, that hooks network-related libc functions\
+  in DYNAMICALLY LINKED programs via a preloaded DLL (dlsym(), LD_PRELOAD)\
+  and redirects the connections through SOCKS4a/5 or HTTP proxies.\
+  It supports TCP only (no UDP/ICMP etc).
+
+  The way it works is basically a HACK; so it is possible that it doesn't\
+  work with your program, especially when it's a script, or starts\
+  numerous processes like background daemons or uses dlopen() to load\
+  "modules" (bug in glibc dynlinker).\
+  It should work with simple compiled (C/C++) dynamically linked programs\
+  though.
+	"
```

Comment:
Start the description right after DESCRIPTION="
Move the spaces at the end of a line before the backslash instead of at the beginning of a new line
Move the last double-quote to the end of the last line

Reply:
https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description